### PR TITLE
Add linux-musl-loongarch64 in rid graphs

### PR DESCRIPTION
**Pull request by am11**
cc @sunlijun-610 @shushanhf

### CODE REVIEW

**Comment by am11** [comment]
FYI @ViktorHofer, they were supposed to be added at the same time linux-musl-riscv64 was added a few months ago. Just noticed this disparity. Also https://github.com/dotnet/sdk/pull/42657, too, is applicable to .net9.0.

File: src/libraries/Microsoft.NETCore.Platforms/src/PortableRuntimeIdentifierGraph.json [hunk 1]
```diff
@@ -249,6 +249,12 @@
           "linux-riscv64"
         ]
       },
+      "linux-musl-loongarch64": {
+        "#import": [
+          "linux-musl",
+          "linux-riscv64"
```
Comment (am11) [inline src/libraries/Microsoft.NETCore.Platforms/src/PortableRuntimeIdentifierGraph.json:255]:
```suggestion
          "linux-loongarch64"
```

File: src/libraries/Microsoft.NETCore.Platforms/src/runtime.json [hunk 2]
```diff
@@ -2135,6 +2135,12 @@
         "linux-riscv64"
       ]
     },
+    "linux-musl-loongarch64": {
+      "#import": [
+        "linux-musl",
+        "linux-riscv64"
```
Comment (am11) [inline src/libraries/Microsoft.NETCore.Platforms/src/runtime.json:2141]:
```suggestion
        "linux-loongarch64"
```

**Review by jkotas — APPROVED** [review]

**Comment by jkotas** [comment]
Thanks!

**Pull request merged**